### DISCUSSION TITLE
Amiga 3000 Kickstart

### DISCUSSION THREAD

**Discussion by adelavig** [BlitterStudio/amiberry#1020] [Q&A] [answered]
Is there an A3000 Kickstart that works with Amiberry? I am trying to get the environment from my old A3000 working on a RasPi 4B but none of the A3000 Kickstarts I have (2.04 and 3.1 copied from the roms on my machine using TransROM and roms from Amiga Forever) will boot, I just get a black screen.

I've tried using the A500/A2000 version roms but I can't emulate a ZorroIII UAE graphics card because those roms don't support ZorroIII.

Thanks.

**Comment by midwan** [comment, marked as answer]
Unfortunately, this is a known issue for now. A3000 doesn't seem to work as expected, probably due to something missing in the emulated hardware. A workaround is to use an A4000 config instead, which should be able to boot your environment.
Keep in mind most of the extra hardware that WinUAE emulates is not implemented in Amiberry (at least, not yet), so you might need to make some changes in your environment.

**Reply by adelavig** [reply]
Thank you. Do you have any plans to include the ZorroII UAE Graphics device so A2000 configs can use RTG?

**Reply by midwan** [reply]
Yes, that should be rather easy as well. It's just that nobody requested it before, and I didn't have to use it myself until now :)

**Reply by midwan** [reply]
I've created this to track that feature request: https://github.com/BlitterStudio/amiberry/issues/1026
